core/src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala

```diff
@@ -67,7 +67,8 @@ import scala.concurrent.duration._
       def cleanUpGreaterThanPeriod(period: Period, name: String): Unit = {
         (metadataCleanupInterval, period) match {
           case (Some(cleanup), Fixed(p)) =>
-            require(cleanup > p, s"$name can not be greater than cleanup-old-persistence-ids")
+            println("comparing " + cleanup + " and " + (p * 0.9) + p)
+            require(cleanup > (p * 1.1), s"$name has to be at least 10% lower than cleanup-old-persistence-ids")
```

Comment:
shouldn't this be `cleanup * 0.9 > p` to match the 10% lower?

Reply:
Yeah that matches the message more accurately. I'll switch.